### PR TITLE
impl(internal/generate): run dotnet container

### PR DESCRIPTION
**Pull request by julieqiu**
Run containers/dotnet from the generator.

### CODE REVIEW

File: internal/generate/run.go [hunk 1]
```diff
@@ -16,13 +16,71 @@ package generate
...
+
+func runDocker(googleapisDir, languageDir, api string) error {
+	if err := runCommand("docker", "build", dotnetContainerDir, "-t", "picard"); err != nil {
+		return err
+	}
+	args := []string{
+		"run",
+		"-v", fmt.Sprintf("%s:/apis", googleapisDir),
+		"-v", fmt.Sprintf("%s:/output", languageDir),
+		dotnetContainerDir,
```
Comment (jskeet) [inline internal/generate/run.go:69]: This should be "picard" instead of the container directory. (It's the tag of the already-built image.)

Comment (julieqiu) [reply internal/generate/run.go]: updated

File: internal/generate/run.go [hunk 2]
```diff
@@ -50,3 +55,32 @@ func cloneLanguageRepo(ctx context.Context, language string) (*gitrepo.Repo, err
 	repoPath := filepath.Join(os.TempDir(), fmt.Sprintf("/google-cloud-%s", language))
 	return gitrepo.CloneOrOpen(ctx, repoPath, languageRepoURL)
 }
+
+const dotnetContainerDir = "containers/dotnet"
+
+func runDocker(googleapisDir, languageDir, api string) error {
+	if err := runCommand("docker", "build", dotnetContainerDir, "-t", "picard"); err != nil {
```
[the remark this answers is not in the document]
Comment (julieqiu) [reply internal/generate/run.go]: updated

File: go.mod [hunk 3]
```diff
@@ -20,10 +20,11 @@ require (
 	github.com/sergi/go-diff v1.3.2-0.20230802210424-5b0b94c5c0d3 // indirect
 	github.com/skeema/knownhosts v1.2.2 // indirect
 	github.com/xanzy/ssh-agent v0.3.3 // indirect
-	golang.org/x/crypto v0.21.0 // indirect
-	golang.org/x/mod v0.12.0 // indirect
-	golang.org/x/net v0.22.0 // indirect
-	golang.org/x/sys v0.18.0 // indirect
-	golang.org/x/tools v0.13.0 // indirect
+	golang.org/x/crypto v0.29.0 // indirect
```
[the remark this answers is not in the document]
Comment (julieqiu) [reply go.mod:23]: happy to talk about this for as long as you want :)

**Review by jskeet — APPROVED** [review]

**Pull request merged**